README.md

```diff
@@ -51,6 +51,11 @@ func main() {
     myLogger.Log("CustomLevelName", "EventName", "message...", map[string]string {
       "key": "value"
     })
+
+    // Level filtering (define per transport!)
+    filteredTransport := logger.NewTransport( ... )
+    filteredTransport.filter = logger.FilterByMinimumLevel(logger.NewLevelFilter("Warning"))
```

Comment:
Yes, that's actually what's happening underneath - the full log entry is passed to the filter, so you can construct a filter based on any of its properties.